packages/cli/src/index.ts

```diff
@@ -25,13 +25,24 @@ program
 program
   .command('encrypt')
   .description('Encrypt and share a secret')
-  .argument('<content>', 'Content to encrypt')
+  .argument('[content]', 'Content to encrypt')
+  .option('-f, --file <file>', 'Path to file to encrypt')
   .option('-p, --password <password>', 'Password to encrypt with')
   .option('-t, --ttl <duration>', 'Time to live (e.g., 1h, 1d)', '1h')
   .option('-b, --burn', 'Burn after reading', true)
   .option('--ip <ip>', 'Restrict access to specific IP address')
   .option('-r, --reads <count>', 'Number of times the secret can be read', undefined)
   .action(async (content, options) => {
+
+    if (options.file && !content) {
```

Comment:
We may want to have an error state for receiving both content and file rather than saying that content overrides the file specification. I prefer erroring rather than the implicit content > file behavior.